capstone_project/src/main/java/com/google/sps/data/Resource.java

```diff
@@ -17,57 +17,45 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
 import java.util.HashMap;
 import java.util.Map;
 
 /** Associates a category, from the Natural Language API taxonomy, to a link to a resource */
 public class Resource {
-  private final DatastoreService datastore;
-  private final String category;
-  private final String resource;
+  private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
-  public Resource() {
-    datastore = DatastoreServiceFactory.getDatastoreService();
-    this.category = "";
-    this.resource = "";
-  }
-
-  public Resource(String category, String resource) {
-    datastore = DatastoreServiceFactory.getDatastoreService();
-    this.category = category;
-    this.resource = resource;
-  }
-
-  /** Returns category */
-  public String getCategory() {
-    return category;
-  }
-
-  /** Returns the resource url associated with the category */
-  public String getResource() {
-    return resource;
-  }
-
-  /** Creates an Entity with this.fields and stores it in Datastore */
-  public void storeResource() {
+  /** Creates a Resource Entity with category and resource properties and stores it in Datastore */
+  public static void storeResource(String category, String resource) {
```

Comment:
is this used anywhere?

Reply:
I now use it in addPreexistingResources(), and it is also used in ResourceTest